src/util/Constants.js

```diff
@@ -35,7 +35,7 @@ const browser = (exports.browser = typeof window !== 'undefined');
  * (or 0 for never)
  * @property {number} [retryLimit=1] How many times to retry on 5XX errors (Infinity for indefinite amount of retries)
  * @property {PresenceData} [presence] Presence data to use upon login
- * @property {WebsocketOptions} [ws] Options for the WebSocket
+ * @property {WebsocketOptions} ws Options for the WebSocket
```

Comment:
q: should we default intents to everything BUT ones that require verification?

Reply:
sounds like a good idea